lua/default-config.lua

```diff
@@ -488,6 +488,28 @@ lvim.lang = {
       },
     },
   },
+  fortran = {
+    formatters = {},
+    linters = {},
+    lsp = {
+      provider = "fortls",
+      setup = {
+        cmd = {
+          DATA_PATH .. "/lspinstall/fortran/venv/bin/fortls",
+        },
+        on_attach = common_on_attach,
+        on_init = common_on_init,
+        capabilities = common_capabilities,
+        root_dir = function(fname)
+          local util = require "lspconfig/util"
+          return util.root_pattern(".fortls", ".git")(fname)
```

Comment:
Is `.git` necessary ?
[Defaults](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/fortls.lua#L8)

Reply:
yes if you want it to activate on `fortran` files without the existence of a `.fortls` setting

Reply:
I agree, but shouldn't this change be on lspconfig ?

Reply:
Sorry I'm not familiar with the project. Would you want me to delete `.git` or move it to somewhere else?

Reply:
It would be perfect it you could remove it from here and add it [there](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/fortls.lua#L8)
Example: https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/sumneko_lua.lua#L10

Reply:
Actually I would also like it to default to current directory like this? Would that make sense here?
```suggestion
          return util.root_pattern(".fortls", ".git")(fname) or vim.fn.getcwd()
```

Reply:
Probably not, but this would be something you could put in your config file :)

Reply:
Ok thank. I have remove all non-default setting.
All my custom config are now in my personal `lvim/config.lua` like:
```
lvim.lang.fortran.lsp.setup.root_dir = function(fname)
  local util = require"lspconfig.util"
  return util.root_pattern(".git", ".fortls")(fname) or vim.fn.getcwd()
end
```
Will try to add this upstream to `lspconfig`.

Reply:
https://github.com/neovim/nvim-lspconfig/pull/1159

Reply:
Thank you very much for working on this and for your flexibility on the implementation !